### ISSUE TITLE
Import/Export page of preferences: accessibility issue

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#5934]
### Bug description

On Windows, using either the Jaws or Narrator screen reader, on the Import/export page of preferences, the names of the group boxes are not read. For the upcoming 3.5 release, this makes it confusing for those want to change the setting in the music imports grouping. The NVDA screen reader does read the group names.

On this page, the radio buttons are grandchildren, rather than children of the group box (which is what they should be). It looks like NVDA can cope with this, but Jaws and Narrator can't.
For the first two group boxes on this page, it looks like an extra panel was introduced to get round a keyboard navigation problem on Macs - see: https://bugzilla.audacityteam.org/show_bug.cgi?id=2692
And I presume that an extra panel was placed in the Music import grouping to fit in with pattern.

My suggestion is that to fix this, these extra panels are removed. This may introduce keyboard navigation issues on Macs, but if this is the case, I think that an alternative workaround needs to be found for Macs which does not cause two of the Windows screen readers to omit important text. In any case, the keyboard navigation on Macs has loads of existing problems which probably already makes it very difficult to use.

@LWinterberg Would you be OK with me creating a PR for this which fixes the problems with Window screen readers by removing these extra panels, but potentially introduces issues on Macs? (Incidentally, on the Track behaviours page of preferences, in the Pasted audio group box there is no extra panel. So here the screen readers don't have any problems, but there may be a keyboard navigation issue on Macs.

### Steps to reproduce

On Windows, with the Narrator screen reader running:
1. Open the preferences dialog
2. Select the Import/Export category
3. Tab through the controls on that page.

### Expected behavior

The names of the group boxes are read.

### Actual behavior

The names of the group boxes are not read.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by DavidBailes** [comment]
> @LWinterberg Would you be OK with me creating a PR for this which fixes the problems with Window screen readers by removing these extra panels, but potentially introduces issues on Macs?

@LWinterberg . A better solution might be to leave the extra panels, but with conditional compilation for only Mac.

**Comment by DavidBailes** [comment]
> A better solution might be to leave the extra panels, but with conditional compilation for only Mac.

I've submitted a PR to do this.
